pkg/route/validation/validation.go

```diff
@@ -354,6 +303,58 @@ func validateTLS(route *routev1.Route, fldPath *field.Path) field.ErrorList {
 	return result
 }
 
+// validateTLSExternalCertificate
+func validateTLSExternalCertificate(ctx context.Context, route *routev1.Route, fldPath *field.Path, sarCreator routecommon.SubjectAccessReviewCreator, secretsGetter corev1client.SecretsGetter) field.ErrorList {
+	tls := route.Spec.TLS
+	var errs field.ErrorList
+
+	errs = append(errs, routecommon.CheckRouteCustomHostSAR(ctx, fldPath, sarCreator)...)
```

Comment:
```suggestion
	errs := append(errs, routecommon.CheckRouteCustomHostSAR(ctx, fldPath, sarCreator)...)
```

Reply:
I think this will cause `undefined: errs` inside append.

Reply:
Yeah, you are right, my bad. You can do something like:

```go
errs := append(field.ErrorList{}, routecommon.CheckRouteCustomHostSAR(ctx, fldPath, sarCreator)...)
```

or simply:
```go
errs := routecommon.CheckRouteCustomHostSAR(ctx, fldPath, sarCreator)
```

Reply:
yeah, updated to
```go
errs := routecommon.CheckRouteCustomHostSAR(ctx, fldPath, sarc)
```